docker/README.md

```diff
@@ -0,0 +1,49 @@
+# FIWARE IoT Agent for the Ultralight 2.0 protocol
+
+[![FIWARE IoT Agents](https://nexus.lab.fiware.org/repository/raw/public/badges/chapters/iot-agents.svg)](https://www.fiware.org/developers/catalogue/)
+[![](https://nexus.lab.fiware.org/repository/raw/public/badges/stackoverflow/iot-agents.svg)](https://stackoverflow.com/questions/tagged/fiware+iot)
+
+An Internet of Things Agent for the Ultralight 2.0 protocol (with
+[AMQP](https://www.amqp.org/), [HTTP](https://www.w3.org/Protocols/) and
+[MQTT](https://mqtt.org/) transports). This IoT Agent is designed to be a bridge
+between Ultralight and the
+[NGSI](https://swagger.lab.fiware.org/?url=https://raw.githubusercontent.com/Fiware/specifications/master/OpenAPI/ngsiv2/ngsiv2-openapi.json)
+interface of a context broker.
+
+It is based on the
+[IoT Agent Node.js Library](https://github.com/telefonicaid/iotagent-node-lib).
+Further general information about the FIWARE IoT Agents framework, its
+architecture and the common interaction model can be found in the library's
+GitHub repository.
+
+This project is part of [FIWARE](https://www.fiware.org/). For more information
+check the FIWARE Catalogue entry for the
+[IoT Agents](https://github.com/Fiware/catalogue/tree/master/iot-agents).
+
```

Comment:
Yes, I'll add an explanatory text of the "How to Use Dockerfile" text into the `README`. I'd prefer not to remove the comment from the `Dockerfile` itself though since a **developer** may not think to look at the `README` 

I know I would tend to look direct at the `Dockerfile` myself. It's nice to find instructions within the one file you are editing.